react-native/react/constants/update.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+
+import type {TypedAction} from '../constants/types/flux'
+import type {Asset, UpdateType} from '../constants/types/flow-types'
+
+export const showUpdatePrompt = 'update:showUpdatePrompt'
+export type ShowUpdateAction = TypedAction<'update:showUpdatePrompt', showUpdatePromptPayload, UpdateError>
+
+export const registerUpdateListener = 'update:registerUpdateListener'
+export type RegisterUpdateListenerAction = TypedAction<'update:registerUpdateListener', any, any>
+
+export const onCancel = 'update:onCancel'
+export type OnCancelAction = TypedAction<'update:onCancel', any, any>
```

Comment:
These should be `TypedAction<'update:onCancel', void, void>` if there is nothing in the payload or error.